File: Python/BookingARoom.py
def main():
    a = input().split()
    rooms = int(a[0])
    booked = int(a[1])
    list = [i+1 for i in range(0,rooms)]
    for i in range(0,booked):
        b = int(input())
        list[b-1] = 0
    if booked >= rooms:
        print("too late")
        return 0
    for i in list:
        if i != 0:
            print(i)
            return 0
main()

File: C++/BookingARoom.cpp
#include <iostream>

using namespace std;

int main() {
  int rooms, booked;
  cin >> rooms >> booked;
  int a[rooms];
  for (int i = 0; i < rooms; i++) {a[i] = i+1;}
  for (int i = 0; i < booked; i++) {
    int b;
    cin >> b;
    a[b-1] = 0;
  }
  if (booked >= rooms) {
    cout << "too late" << endl;
    return 0;
  }
  for (int i = 0; i < rooms; i++) {
    if (a[i] != 0) {
      cout << a[i] << endl;
      return 0;
    }
  }
}
